### ISSUE TITLE
Spot training with HyperparameterTuner

### ISSUE THREAD

**Issue by austinmw** [aws/amazon-sagemaker-examples#869]
I have a `PyTorch` estimator, where I've set:

```python
train_use_spot_instances=True
train_max_run=3600
train_max_wait=7200
```
I then tried to create a `HyperparameterTuner` with my estimator, but got the error:

> ClientError: An error occurred (ValidationException) when calling the CreateHyperParameterTuningJob operation: Invalid MaxWaitTimeInSeconds. It is only supported when EnableManagedSpotTraining is set to true

I don't see any documentation for spot training + hyperparameter tuning. How should this be configured?

**Comment by rmwenzel** [comment]
Ran into the same issue with `Tensorflow` estimator.

**Comment by rmwenzel** [comment]
Turns out it was a bug, fixed in V1.39: aws/sagemaker-python-sdk/issues/1011

**Comment by hsl89** [comment]
Dear SageMaker Community,
                 In our attempts to provide this repository with a better level of support                 going forward, we’re closing issues that were opened prior to the v2 release of the SDK.                 This is because we believe that over time many of the issues                 posted were solved with the latest release or other recent changes to the repo.                 This will help us reallocate resources towards issues that are                more likely to still be relevant today.                Some of the issues experienced now can be resolved by referencing the                v2 guide: https://sagemaker.readthedocs.io/en/stable/v2.html                 In this guide, you can find simple solutions to common notebook errors,                 like the renaming of parameters and classes.                If you believe your issue is still ongoing and you have updated error                messaging or other info, please re-open it, and we investigate the issue.
                Best Regards,
                AWS SageMaker Team
